### ISSUE TITLE
Message below input - DAC_Usability_Errors_Issue1

### ISSUE THREAD

**Issue by gtvj** [nationalarchives/ds-wagtail#186]
On page 56 of the report DAC describe a usability issue relating to error messages being placed below inputs. 

DAC have suggested we place the message above the input so that users encounter the message before the input. 

### Why was this not picked up in the in-house testing? 
While this is not a WCAG failure we were aware of potential issues relating to the message appearing below the input. We had used `aria-describedBy` so that users would be presented with the message when they encounter the input. DAC instead recommend providing this information before the user encounters the input.
